.github/workflows/go.yml

```diff
@@ -0,0 +1,28 @@
+name: Go
+on: [push]
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.13
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.13
+      id: go
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v1
+
+    - name: Get dependencies
+      run: |
+        go get -v -t -d ./...
+        if [ -f Gopkg.toml ]; then
```

Comment:
Why do we need this?

Reply:
This whole file was copied from the Github Actions golang setup, and I haven't edited it yet.

Which is to say: 🤷‍♀

Reply:
I think we can remove this :)

Reply:
To be super clear: this is a draft PR, I literally just copy pasted all of this. Pleeeeeease wait until it's ready before making comments.